res/flow-capture.yml

```diff
@@ -17,6 +17,8 @@ spec:
       serviceAccountName: netobserv-cli
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
+      tolerations:
+      - operator: Exists
```

Comment:
what does it mean to have just `operator: Exists` without a key?

Reply:
(btw, how does that work in the operator? I never realized we did anything special to have the agents on the master nodes, do we?)

Reply:
That's exactly how it's done on the operator actually:
https://github.com/netobserv/network-observability-operator/blob/1aaeb6b3c0719c9ef73667daa3a04df0d69fd3ba/pkg/helper/flowcollector.go#L227

It's a trick to skip taints:
```
// Operator represents a key's relationship to the value.
// Valid operators are Exists and Equal. Defaults to Equal.
// Exists is equivalent to wildcard for value, so that a pod can
// tolerate all taints of a particular category.
// +optional
```

Reply:
Ok, I learned something, thanks! :-)
Wondering if we should make it an option? Like, if we start capturing pcap on master nodes, wouldn't be a higher risk to make the whole cluster unstable?

Reply:
You can use the `--node-selector` option to filter out which node to target. I feel it's good enough.
https://github.com/netobserv/network-observability-cli/pull/146 is adding a example with it